metric_learn/_util.py

```diff
@@ -337,18 +364,34 @@ def transformer_from_metric(metric):
   just return its elementwise square root (since the diagonalization of
   the matrix is itself).
 
+  Parameters
+  ----------
+  metric : symmetric `np.ndarray`, shape=(d x d)
+    The input metric, from which we want to extract a transformation matrix.
+
+  tol : positive float, optional
+    Eigenvalues of `metric` between 0 and - tol are considered zero. If tol is
+    None, and w are `metric`'s eigenvalues, and eps is the epsilon value for

```

Comment:
same as above

Reply:
How to explain how the tolerance is set here then without talking about the eigenvalues ? Though maybe an explanation talking about the highest eigenvalue could be better ? Like:
```
    Eigenvalues of `metric` between 0 and - tol are considered zero. If tol is
    None, and w_max is `metric`'s largest eigenvalue, and eps is the epsilon
    value for datatype of w, then tol is set to w_max * metric.shape[0] * eps.
```

Reply:
nevermind, indeed here we need to mention eigenvalues
the suggested reformulation looks good